<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kia_debug_commands</class>
 <widget class="QDialog" name="Kia_debug_commands">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Название для БД(англ.)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Подадресс</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cb_sub_address"/>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="cb_word_data"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Направление</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="le_struct_id"/>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="le_struct_id_desc"/>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cb_direction"/>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="cb_format"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Число СД</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Название для протокола</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Формат</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="3">
    <widget class="QListWidget" name="lw_commands"/>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="delete_command">
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QTableWidget" name="tw_for_sd"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="add_command">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
